skt/reporter.py

```diff
@@ -333,8 +333,9 @@ def getmergefailure(self):
 
     def getbuildfailure(self):
         attname = "build.log"
-        result = ["\n-----------------------",
-                  "Build failed: see attached %s" % attname]
+        result = ['However, the build failed. We are attaching the build '
+                  'output for',
+                  'you to review (build.log).']
```

Comment:
Oh, and I think "for you to review" is also a no-go. We cannot ask people to review any of our results. We're offering help.

Reply:
Will change it to "... attaching the build output for more information"